_posts/2021-01-09-Julia-1.6-what-has-changed-since-1.0.md

```diff
@@ -0,0 +1,639 @@
+---
+title: "Julia 1.6: what has changed since Julia 1.0?"
+layout: default
+tags:
+   - julia
+---
+
+Julia 1.0 came out well over 2 year ago.
+Since then a lot has changed; and a lot hasn't.
+Since Julia 1.0 was a commitment to no breaking changes;
+but that is not to say no new features have been added to the language.
+
+<!--more-->
+
+Julia 1.6 is the first feature release since 1.0.
+Prior to 1.0 all releases were feature releases; they came out when they were ready.
+Since then all releases (except 1.6) have been timed releases.
+The release candidate is cut from the main branch every 3 months; and they are released after an [extensive round of additional checking](https://github.com/JuliaLang/julia/pull/35846), which often takes weeks to months.
+Julia 1.6 was soft-slated to be the new long-term-support (LTS) version that would have bug-fixes backported to it for the next few years.
+The current LTS is julia 1.0, which has now had 5 patch releases made.
+Since it was going to be supported for a long-time people wanted to make sure everything good got in; thus it was a feature release.
+The core developers have demurred on if 1.6 will actually be selected to be the new LTS (even if it wasn't it won't assend to being LTS til it stops being the current Stable).
+
+My impression now is that they feel like it has too many cool new things; and that a few things didn't quite make it in even with the extended release cycle.
+So it's looking likely to me that 1.7 will actually be the LTS; but that it might also be a feature release -- possibly this time a much shorter release period than usual.
+In practice I think for a lot of package maintainer's 1.6 will be a LTS, in that that is the oldest version they will make sure to continue to support.
+There has been too many cool new things (as this post will detail) to stay back to only 1.0 features.
+Already a lot of packages have dropped support for julia versions older than 1.3.
```

Comment:
```suggestion
Already a lot of packages have dropped support for Julia versions older than 1.3.
```
Perhaps mention something along the lines of "due to Artifacts" (since I think that's the main reason)? Or perhaps  I will see that later in the post.